uwsgi_configurations.ini

```diff
@@ -0,0 +1,12 @@
+[uwsgi]
```

Comment:
A really nitpicky thing, but why is this uwsgi_configuration*s* ? Usually plurals are not used with filenames.

Reply:
Also uwsgi configuration probably should not be in the root. It is not really part of the linkedevents proper. That's why I stuffed it into /deploy.

Reply:
> A really nitpicky thing, but why is this uwsgi_configurations ? Usually plurals are not used with filenames.

No particular reason, in my mind I just read it as "uwsgi configurations file". Done in 956425330192f99b1f1442add9248a519582f19a

> Also uwsgi configuration probably should not be in the root. It is not really part of the linkedevents proper. That's why I stuffed it into /deploy.

That's right, forgotten about the fact that even [according to KuVa's DevOps best practices](https://helsinkisolutionoffice.atlassian.net/wiki/spaces/DD/pages/73859073/Dockerfile+guidelines#.prod%2F), I should have put it in `.prod` folder at the root. 

I think it'd be better if it was named `.server` or something based on our own discussion; I kept the `.prod` folder name to be consistent with the convention in place. Is that fine with you? 😊

Reply:
Sure. I'll complain about the "prod" thing in other forums